### PR TITLE
Commit first swagger version

### PR DESCRIPTION
**Pull request by KenKLai**
Commit first swagger version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+#### `npm start`
+Starts the development server.
+
+#### `npm run build`
+Bundles the spec and prepares web_deploy folder with static assets.
+
+#### `npm test`
+Validates the spec.
+
+#### `npm run gh-pages`
```
Comment (anthonyng-melco) [inline README.md]: We don't need this section because we won't use Travis CI to deploy

File: spec/openapi.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Example.com
```
Comment (anthonyng-melco) [inline spec/openapi.yaml]: This repo is for developers who wish to integrate the VIK API. I suggest the title `Service API swagger for G2 VIK`

File: spec/openapi.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  termsOfService: 'https://example.com/terms/'
```
Comment (anthonyng-melco) [inline spec/openapi.yaml]: I think we don't need it

File: spec/openapi.yaml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  contact:
+    email: contact@example.com
```
Comment (anthonyng-melco) [inline spec/openapi.yaml]: I would suggest to use the DX engineer team email `DLITDXEngineering@melco-resorts.com`

File: spec/openapi.yaml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    url: 'http://example.com/contact'
```
Comment (anthonyng-melco) [inline spec/openapi.yaml]: We don't have it

File: spec/openapi.yaml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+  x-logo:
+    url: 'https://apis.guru/openapi-template/logo.png'
+  description: >
+    This is an **example** API to demonstrate features of OpenAPI specification
```
Comment (anthonyng-melco) [inline spec/openapi.yaml]: Please add a descriptive title

File: spec/openapi.yaml [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+    # Introduction
```
Comment (anthonyng-melco) [inline spec/openapi.yaml]: We don't need this section

File: spec/openapi.yaml [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+    implementation logic. Similar to what
+
+    interfaces have done for lower-level programming, OpenAPI removes the
+    guesswork in calling the service.
+externalDocs:
+  description: Find out how to create Github repo for your OpenAPI spec.
+  url: 'https://github.com/Rebilly/generator-openapi-repo'
+tags:
+  - name: patron-mgnt-api-controller
+    description: ''
```
Comment (anthonyng-melco) [inline spec/openapi.yaml]: Please add a description

File: spec/openapi.yaml [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+  - name: tiers-controller
+    description: ''
```
Comment (anthonyng-melco) [inline spec/openapi.yaml]: Please add a description

File: spec/components/schemas/Result_TiersConfig_.yaml [hunk 10]
```diff
@@ -0,0 +1,9 @@
+type: object
```
Comment (anthonyng-melco) [inline spec/components/schemas/Result_TiersConfig_.yaml]: Is there any reason for the file name "Result_TiersConfig_.yaml" with an underscore suffix?

File: spec/components/securitySchemes/api_key.yaml [hunk 11]
```diff
@@ -0,0 +1,3 @@
+type: apiKey
```
Comment (anthonyng-melco) [inline spec/components/securitySchemes/api_key.yaml]: There is no need to use API KEY

File: spec/components/securitySchemes/basic_auth.yaml [hunk 12]
```diff
@@ -0,0 +1,2 @@
+type: http
```
Comment (anthonyng-melco) [inline spec/components/securitySchemes/basic_auth.yaml]: There is need to do authentication

**Comment by KenKLai** [comment]
commit new updates for review comments

**Pull request merged**